tests/SprykerSdkTest/Spryk/Model/Spryk/Definition/Argument/Resolver/ArgumentDefinitionReturnTest.php

```diff
@@ -0,0 +1,163 @@
+                    'Bar',
+                ],
+            ],
+        ];
+
+        // Act
+        foreach ($arguments as $argumentName => $argumentDefinition) {
+            $normalizedArguments[$argumentName] = $argumentDefinitionNormalizer->normalizeArgumentDefinition($argumentDefinition);
+        }
+
+        // Assert
+        $this->assertSame($expectedResult, $normalizedArguments);
+    }
+
+    /**
+     * @return void
+     */
+    public function testNormalizeArgumentDefinitionReturnsCorrectValueWithArgumentWithoutValueKey(): void
+    {
+        // Arrange
+        $argumentDefinitionNormalizer = $this->getArgumentDefinitionNormalizer();
+        $normalizedArguments = [];
+
+        $arguments = [
+            'organization' => 'Pyz',
+            'mode' => 'project',
+            'body' => [
+                'Foo',
+                'Bar',
+            ],
+        ];
+
+        $expectedResult = [
+            'organization' => [
+                'value' => 'Pyz',
+            ],
+            'mode' => [
+                'value' => 'project',
+            ],
+            'body' => [
+                'value' => [
+                    'Foo',
+                    'Bar',
+                ],
+            ],
+        ];
+
+        // Act
+        foreach ($arguments as $argumentName => $argumentDefinition) {
+            $normalizedArguments[$argumentName] = $argumentDefinitionNormalizer->normalizeArgumentDefinition($argumentDefinition);
+        }
+
+        // Assert
+        $this->assertSame($expectedResult, $normalizedArguments);
+    }
+
+    /**
+     * @return void
+     */
+    public function testNormalizeArgumentDefinitionReturnsCorrectValueWithArgumentWithValueKeyAndWithOtherConfiguration(): void
+    {
+        // Arrange
+        $argumentDefinitionNormalizer = $this->getArgumentDefinitionNormalizer();
+        $normalizedArguments = [];
+        $arguments = [
+            'organization' => [
+                'inherit' => 1,
+                'value' => 'Pyz',
+            ],
+            'mode' => [
+                'value' => 'project',
+            ],
+        ];
+
+        $expectedResult = [
+            'organization' => [
+                'inherit' => 1,
+                'value' => 'Pyz',
+            ],
+            'mode' => [
+                'value' => 'project',
+            ],
+        ];
+
+        // Act
+        foreach ($arguments as $argumentName => $argumentDefinition) {
+            $normalizedArguments[$argumentName] = $argumentDefinitionNormalizer->normalizeArgumentDefinition($argumentDefinition);
+        }
+
+        // Assert
+        $this->assertSame($expectedResult, $normalizedArguments);
+    }
+
+    /**
+     * @return \SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Normalizer\ArgumentDefinitionNormalizerInterface
+     */
+    protected function getArgumentDefinitionNormalizer(): ArgumentDefinitionNormalizerInterface
+    {
+        return new ArgumentDefinitionNormalizer();
```

Comment:
Please use the container to get the service. `\SprykerSdkTest\Module\SprykHelper::getClass()` Otherwise we would need to change the test when we inject dependencies in the future.